cmd/tailscale/cli/debug.go

```diff
@@ -65,7 +65,7 @@ var debugCmd = &ffcli.Command{
 		{
 			Name:      "component-logs",
 			Exec:      runDebugComponentLogs,
-			ShortHelp: "enable/disable debug logs for a component",
+			ShortHelp: "enable/disable debug logs for a component (run with no args for list of components)",
```

Comment:
Unfortunately this is a string literal so can't be dynamically generated, hence the awkward "run with no args" instruction. Better solutions welcome!